Index: src/main/java/model/deck/BuildingDeck.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package model.deck;\n\nimport model.cards.buildingCards.BuildingCard;\nimport model.enums.Era;\n\nimport java.util.List;\n\npublic class BuildingDeck {\n    private Era era;\n    private List<BuildingCard> cards;\n\n    public boolean isEmpty() {\n        return cards.isEmpty();\n    }\n\n    //setup\n    public void initializeDeck(int playerCount){\n    }\n    public BuildingDeck(Era era){}\n    public void addCard(BuildingCard card)\n\n    //access\n    // used at the beginning of each Era to populate the top row with the Building cards of the current Era\n    public List<BuildingCard> drawAll() {\n        return cards;\n    }\n\n    public Era getEra(){return era;}\n}\n
===================================================================
diff --git a/src/main/java/model/deck/BuildingDeck.java b/src/main/java/model/deck/BuildingDeck.java
--- a/src/main/java/model/deck/BuildingDeck.java	(revision a7e6df9663e6e0b6a2e0eb9b50cc27c16ff06aa4)
+++ b/src/main/java/model/deck/BuildingDeck.java	(date 1774698364320)
@@ -17,7 +17,7 @@
     public void initializeDeck(int playerCount){
     }
     public BuildingDeck(Era era){}
-    public void addCard(BuildingCard card)
+    public void addCard(BuildingCard card){}
 
     //access
     // used at the beginning of each Era to populate the top row with the Building cards of the current Era
